capstone/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Mon Jun 22 17:44:04 UTC 2020
+#Wed Jun 24 07:26:49 UTC 2020
```

Comment:
@anikabagga @himaniyadav @lxcyqx Is this a file that needs to be updated? Could it be added to the `.gitignore`?

Reply:
Could you all look together through the list of files that are being added and make sure gitignore any that aren't necessary?

Reply:
Yeah I've added target/ to the gitignore but I think some files had been added already prior to me doing that. So we should go back and delete everything under target/.

Reply:
removed target folder